package.json

```diff
@@ -348,18 +316,6 @@
         "category": "Tekton",
         "enablement": "tekton:tkn"
       },
-      {
-        "command": "tekton.pipelinerun.delete",
-        "title": "Delete",
-        "category": "Tekton",
-        "enablement": "tekton:tkn"
-      },
-      {
-        "command": "tekton.pipelinerun.delete.palette",
```

Comment:
this will remove the functionality for deleting pipelinerun from the command palette.

Reply:
and we have other commands also for delete operation from the command palette which is being removed.

Reply:
We already discuss that.